bundles/binding/org.openhab.binding.xpl/src/main/java/org/openhab/binding/xpl/internal/XplGenericBindingProvider.java

```diff
@@ -159,6 +158,7 @@ else if (configParts[1].equalsIgnoreCase("COMMAND"))
 	    } 
 		
 		addBindingConfig(item, config);
+		super.processBindingConfiguration(context, item, bindingConfig);
 	}
```

Comment:
Moved super.processBindingConfiguration to end of override, since previous steps could fail and we don't want invalid binding config objects in the system that were rejected.